README.md

```diff
@@ -110,13 +109,11 @@ From 2019-02-01 to 2019-03-31 : 13h1m48s
 
 
 ## Why did I make it?
-https://github.com/kimmobrunfeldt/git-hours
-
-To install the git hours command through the above repository uri, you need to install node.js and dependencies.
-I thought that it would be good if there was a tool that can be executed immediately after downloading.
-
-reference model : https://github.com/kimmobrunfeldt/git-hours#how-it-works
-
-The code I wrote before the first commit was set to a model that basically add 1 hour, including programming immersion time.
-
-I like projects that are carried out by telecommuting. For me, it is more convenient than https://toggl.com service. It is also a trust model compared to the toggle service.
+I interest working from home space.
+I wanted to know how to get working time.
+I looked like a [toggle-style program](https://toggl.com) was too annoying. because the user has to check every time the work starts or ends.
+[kimmobrunfeldt's git-hours](https://github.com/kimmobrunfeldt/git-hours#how-it-works) command model approach looked good to me.
+I installed it via https://github.com/kimmobrunfeldt/git-hours, but it didn't install well on my computer. It was because of the node.js dependency.
+So, I recreated command with Go language in a similar way.
+Also, I added some of the features I needed to create a estimate sheet.
```

Comment:
- some features
- features that help to create an estimate sheet
- an + 'e'stimate